OJP_Trips.xsd

```diff
@@ -582,6 +586,11 @@
 					</xs:sequence>
 				</xs:complexType>
 			</xs:element>
+			<xs:element name="LineIdAtAlighting" type="xs:normalizedString" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Identifier of the alighted line/service at the alighting. This is not a reference. This identifier is used to recognize in a distributed environment, that two systems refer to the same line service (or service) while using their own internal references.</xs:documentation>
```

Comment:
> Identifier of the alighted line/service _during_ alighting. This is not a reference. This identifier is used to recognize _the line_ in a distributed environment _(two_ systems _referring_ to the same line service (or service) while using their own internal references).

Reply:
Um, is your proposal to take over your edited text? "Line service (or service)" sounds really weird, simply "service" would be correct.